gbpservice/tests/contrib/devstack/local-nfp.conf

```diff
@@ -6,6 +6,9 @@ RABBIT_PASSWORD=abc123
 SERVICE_PASSWORD=$ADMIN_PASSWORD
 SERVICE_TOKEN=abc123
 
+ENABLE_NFP=True
+NFP_DEVSTACK_MODE=advanced
```

Comment:
Lets submit with base mode first. NFP_DEVSTACK_MODE=base
Change GBPSERVICE_REPO and GBPSERVICE_BRANCH below accordingly